src/neo/Plugins/Plugin.cs

```diff
@@ -116,7 +116,21 @@ public virtual void Dispose()
 
         protected IConfigurationSection GetConfiguration()
         {
-            return new ConfigurationBuilder().AddJsonFile(ConfigFile, optional: true).Build().GetSection("PluginConfiguration");
+            IConfigurationSection config = null;
+            int remainingTimes = 3;
+            while (remainingTimes > 0 && config == null)
+            {
+                try
+                {
+                    config = new ConfigurationBuilder().AddJsonFile(ConfigFile, optional: true).Build().GetSection("PluginConfiguration");
+                }
+                catch (FormatException)
+                {
+                    remainingTimes--;
+                    Thread.Sleep(10);
+                }
+            }
+            return config;
```

Comment:
if `config` is null in line 133, we should throw an exception

Reply:
Have changed correspondingly